src/assets/products/microsoft.json

```diff
@@ -1,4 +1,13 @@
 [
+	{
+		"dateClose": "2011-03-16",
+		"dateOpen": "2004-04-01",
+		"description": "Windows Live Spaces was a blogging platform and social network service",
+		"link": "https://en.wikipedia.org/wiki/Windows_Media_Center",
```

Comment:
@MIfoodie, I think this should be https://en.wikipedia.org/wiki/Windows_Live_Spaces or similar right? :)

Reply:
@edvinlinden Oops I was copying and pasting the format and forgot to change the link. My bad!